backend/src/src-wsimport/src/main/java/org/eclipse/sw360/wsimport/domain/WsProjectLibs.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Verifa Oy, 2018. Part of the SW360 Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.sw360.wsimport.domain;
+
+/**
+ * @author: ksoranko@verifa.io
+ */
+public class WsProjectLibs {
```

Comment:
I am not sure can we just remove these classes, because the response for getProjetcVitals is like:

``` java
{
  "projectVitals": [
    {
      "id": 371829,
      "name": "POCT1",
      "token": "tokentokentoken",
      "creationDate": "2018-01-18 11:04:55 +0000",
      "lastUpdatedDate": "2018-01-23 11:59:26 +0000"
    }
  ]
}
```
, so it is a list but it has this identifier. Do you know how to deal with this, because just using
``` java
WsProjectVitalInformation[] wsProjectVitals = null;
        try {
            wsProjectVitals = gson.fromJson(projectVitalString, WsProjectVitalInformation[].class);
...
``` 
in WsProjectService will not work?